<launch>

    <!-- parametre -->
    <arg name="robot_namespace" default="my_gen3_lite" />

    <!-- launch turtbo script -->
    <node name='kinova_service_client' pkg='kinova' type='service_client.py' ns="$(arg robot_namespace)" output="screen" />
</launch>
